routes/weatherProviders/Apple.ts

```diff
@@ -15,6 +15,7 @@ export default class AppleWeatherProvider extends WeatherProvider {
 		super();
 
         if (!process.env.APPLE_PRIVATE_KEY) {
+            return;
 			throw "APPLE_PRIVATE_KEY environment variable is not defined.";
```

Comment:
Shouldn't this line be removed? It is unreachable code with the return statement above it and adds confusion when scanning the code.

Reply:
This line is added intentionally to address issue #160